### ISSUE TITLE
Incorrect tags applied to package

### ISSUE THREAD

**Issue by kichik** [cdklabs/construct-hub#901]
### What is the URL of the page with the issue?

https://constructs.dev/packages/@cloudsnorkel/cdk-github-runners/v/0.0.11?lang=typescript

### What did you do?

I created a package with specific keywords and a readme that mentions others.

### What did you expect to happen?

I expected only my keywords to be used as tags.

### What actually happened?

My keywords were used as tags, but also keywords automatically found in `README.md`. Specifically it found Kubernetes because I mentioned another package that provides the same solution but for Kubernetes. My package doesn't use Kubernetes. It also found MySQL from the sample code even though the package doesn't have much to do with MySQL.

### Web Browser

Chrome

**Comment by kichik** [comment]
Makes total sense. The categorization does help constructs hub. Maybe we can have a special tag or other way to prevent this? A special comment line in the readme?

**Comment by Chriscbr** [comment]
Yep - it's definitely possible to add some functionality that disables this behavior by adding a custom tag to the package, like `construct-hub/do-not-autotag` (and we'd probably want to hide this meta-tag on the front-end too). PRs are welcome.
